config/rbac/role.yaml

```diff
@@ -5,219 +5,62 @@ metadata:
   creationTimestamp: null
```

Comment:
I don't think generation of that file work correctly at the moment. RBAC for MCAD has to be declared for it to work.

Reply:
Good catch. I may just remake this by hand

Reply:
Yes, manual edit is fine at the moment.